firmware/user/display/oled.c

```diff
@@ -322,70 +303,35 @@ bool ICACHE_FLASH_ATTR setOLEDparams(bool turnOnOff)
     return (0 == cnlohr_i2c_end_transaction());
 }
 
-/**
- * @brief Find the first and last differences in a page
- *
- * @param prior The prior framebuffer to compare
- * @param curr  The current framebuffer to compare
- * @param bounds A pointer to return the first and last difference index through
- * @return true if there are any differences, false otherwise
- */
-inline bool ICACHE_FLASH_ATTR findDiffBounds(uint8_t* prior, uint8_t* curr, int16_t* bounds)
+int ICACHE_FLASH_ATTR updateOLEDScreenRange( uint8_t minX, uint8_t maxX, uint8_t minPage, uint8_t maxPage )
 {
-    int16_t col;
-    bool anyDiffs = false;
-    // Look for the first difference
-    for (col = 0; col < SSD1306_NUM_COLS; col++)
+    uint8_t encountered_error = false;
+    uint8_t x, page;
+
+    setColumnAddress( minX, maxX );
+    setPageAddress( minPage, maxPage );
+
+    if( cnlohr_i2c_end_transaction() ) encountered_error = true;
```

Comment:
Where is the equivalent `cnlohr_i2c_start_transaction()`? The only one I see in `setOLEDparams()`

Reply:
Oh, you're calling `SendStart()` directly. Is that really necessary?

Reply:
Did this work?  Did I successfully update the PR before acceptance?  It looks like I did?